### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by scriptnull**

### CODE REVIEW

File: external/feature.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Purus in mollis nunc sed. Diam ut venenatis tellus in metus vulputate. Tristique et egestas quis ipsum suspendisse ultrices. Platea dictumst vestibulum rhoncus est pellentesque. Eu non diam phasellus vestibulum lorem sed risus. Semper risus in hendrerit gravida. Pharetra massa massa ultricies mi quis hendrerit dolor magna. Urna id volutpat lacus laoreet. Lacus suspendisse faucibus interdum posuere lorem. Mauris pharetra et ultrices neque ornare. Sed lectus vestibulum mattis ullamcorper velit sed. Non odio euismod lacinia at. In mollis nunc sed id. Molestie ac feugiat sed lectus vestibulum mattis ullamcorper velit. Gravida cum sociis natoque penatibus et magnis. Tincidunt dui ut ornare lectus. Integer quis auctor elit sed.
```
Comment (scriptnull) [inline external/feature.md:1]: comment on first paragraph

Comment (scriptnull) [reply external/feature.md:1]: second comment on the first para.

This is a multi line comment.

Comment (scriptnull) [reply external/feature.md:1]: another comment

Comment (scriptnull) [reply external/feature.md:1]: checking more

Comment (scriptnull) [reply external/feature.md:1]: some more comments but this time it is multi line
line 2
line 3
line 4

line6

**Comment by scriptnull** [comment]
commenting on the pr

**Comment by scriptnull** [comment]
checking if event doesn't skip

**Comment by scriptnull** [comment]
check shadow

**Comment by scriptnull** [comment]
ping

File: external/feature.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+Aliquam etiam erat velit scelerisque in. Rhoncus est pellentesque elit ullamcorper dignissim cras tincidunt lobortis feugiat. Vitae tortor condimentum lacinia quis. In dictum non consectetur a erat nam at lectus urna. Tristique risus nec feugiat in fermentum posuere urna nec. Orci porta non pulvinar neque laoreet. Tincidunt dui ut ornare lectus sit amet est placerat. Pretium lectus quam id leo in vitae turpis. Facilisis sed odio morbi quis commodo odio aenean. Vitae tortor condimentum lacinia quis vel eros. Viverra ipsum nunc aliquet bibendum. Iaculis nunc sed augue lacus viverra. Auctor elit sed vulputate mi. Egestas maecenas pharetra convallis posuere morbi leo urna molestie. Iaculis at erat pellentesque adipiscing commodo elit at.
```
Comment (scriptnull) [inline external/feature.md:7]: hello

Comment (scriptnull) [reply external/feature.md:7]: yo

Comment (scriptnull) [reply external/feature.md:7]: check

Comment (scriptnull) [reply external/feature.md:7]: log

Comment (scriptnull) [reply external/feature.md:7]: space

File: external/feature.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Eu sem integer vitae justo eget magna fermentum. Nunc aliquet bibendum enim facilisis gravida neque. Proin nibh nisl condimentum id. Suspendisse interdum consectetur libero id. Quis auctor elit sed vulputate mi sit amet. Laoreet id donec ultrices tincidunt arcu non sodales neque. In hac habitasse platea dictumst vestibulum rhoncus est pellentesque. Lorem dolor sed viverra ipsum nunc. Urna duis convallis convallis tellus id interdum velit laoreet. Volutpat diam ut venenatis tellus in metus vulputate eu. Lorem ipsum dolor sit amet consectetur adipiscing. Nisl purus in mollis nunc sed id semper risus.
+
```
Comment (scriptnull) [inline external/feature.md:4]: another conversation

Comment (scriptnull) [reply external/feature.md:4]: reply

File: external/feature.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+Elementum nisi quis eleifend quam. Id donec ultrices tincidunt arcu. Cursus turpis massa tincidunt dui ut. Magna eget est lorem ipsum dolor. Sodales ut etiam sit amet nisl purus in mollis nunc. Placerat in egestas erat imperdiet sed euismod. Fringilla est ullamcorper eget nulla facilisi etiam dignissim diam quis. Vitae ultricies leo integer malesuada nunc vel. Vitae elementum curabitur vitae nunc sed velit dignissim sodales ut. Tortor condimentum lacinia quis vel eros donec ac odio. Ullamcorper velit sed ullamcorper morbi tincidunt ornare massa eget egestas. Tincidunt praesent semper feugiat nibh sed pulvinar proin gravida hendrerit. Duis at consectetur lorem donec massa sapien faucibus et molestie. Ut pharetra sit amet aliquam id diam maecenas. Sit amet luctus venenatis lectus magna fringilla urna porttitor. Sed nisi lacus sed viverra tellus in hac. Nisl rhoncus mattis rhoncus urna neque viverra justo nec. Eu ultrices vitae auctor eu augue. Non pulvinar neque laoreet suspendisse interdum consectetur libero id. Nibh mauris cursus mattis molestie a iaculis at erat.
```
Comment (scriptnull) [inline external/feature.md:9]: super comment

Comment (scriptnull) [reply external/feature.md:9]: reply comment

Comment (scriptnull) [reply external/feature.md:9]: c

File: external/feature.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+Neque egestas congue quisque egestas diam in arcu cursus. Risus nullam eget felis eget nunc. Id aliquet lectus proin nibh nisl condimentum id venenatis. Hendrerit gravida rutrum quisque non tellus. Amet tellus cras adipiscing enim eu turpis egestas pretium aenean. Erat nam at lectus urna duis convallis convallis tellus id. Nunc non blandit massa enim nec dui nunc. Nullam vehicula ipsum a arcu cursus. Elementum pulvinar etiam non quam lacus suspendisse. Nullam eget felis eget nunc lobortis mattis. Metus dictum at tempor commodo ullamcorper. Feugiat in ante metus dictum at tempor. Varius duis at consectetur lorem. Pretium vulputate sapien nec sagittis aliquam malesuada. Praesent elementum facilisis leo vel. Libero id faucibus nisl tincidunt eget nullam non nisi est. Vestibulum morbi blandit cursus risus at ultrices mi tempus imperdiet. Pulvinar elementum integer enim neque volutpat ac tincidunt. Fermentum odio eu feugiat pretium nibh ipsum. Enim praesent elementum facilisis leo vel.
+
```
Comment (scriptnull) [inline external/feature.md:6]: new line

Comment (scriptnull) [reply external/feature.md:6]: reply comment

**Review by gh-team-member — APPROVED** [review]
accept

**Review by gh-team-member — APPROVED** [review]
approve again

**Review by gh-team-member — APPROVED** [review]
new review

File: feature-on-root.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Venenatis tellus in metus vulputate eu scelerisque felis imperdiet. Porttitor rhoncus dolor purus non. Quam viverra orci sagittis eu volutpat odio. Tempus egestas sed sed risus pretium quam vulputate dignissim suspendisse. Et odio pellentesque diam volutpat commodo sed egestas egestas. Risus in hendrerit gravida rutrum quisque non. Erat velit scelerisque in dictum non. Massa tincidunt dui ut ornare lectus sit. Volutpat lacus laoreet non curabitur gravida. Mi ipsum faucibus vitae aliquet nec ullamcorper sit amet risus. Sit amet mattis vulputate enim nulla aliquet porttitor lacus luctus. Euismod in pellentesque massa placerat duis. Ullamcorper sit amet risus nullam eget felis eget nunc. Phasellus faucibus scelerisque eleifend donec pretium vulputate sapien. Metus aliquam eleifend mi in. Tortor condimentum lacinia quis vel eros donec. Sollicitudin tempor id eu nisl nunc mi ipsum. Viverra adipiscing at in tellus integer feugiat. Dui id ornare arcu odio ut sem nulla pharetra diam.
+
```
Comment (gh-team-member) [inline feature-on-root.md:2]: change 1

File: feature-on-root.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+Pellentesque pulvinar pellentesque habitant morbi tristique senectus et. Luctus venenatis lectus magna fringilla urna porttitor rhoncus. Laoreet suspendisse interdum consectetur libero id faucibus. In mollis nunc sed id semper risus in hendrerit gravida. Fringilla ut morbi tincidunt augue interdum velit euismod. Orci nulla pellentesque dignissim enim sit amet. Egestas sed sed risus pretium quam vulputate dignissim suspendisse in. Eu turpis egestas pretium aenean pharetra. Sed sed risus pretium quam vulputate dignissim suspendisse. Augue ut lectus arcu bibendum. Scelerisque varius morbi enim nunc faucibus a pellentesque sit.
```
Comment (gh-team-member) [inline feature-on-root.md:3]: change 2

**Review by gh-team-member — CHANGES_REQUESTED** [review]
Please make the following changes.

File: external/feature.md (hunk 1 above)
Comment (gh-team-member) [inline external/feature.md:1]: c1

File: external/feature.md [hunk 8]
```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Purus in mollis nunc sed. Diam ut venenatis tellus in metus vulputate. Tristique et egestas quis ipsum suspendisse ultrices. Platea dictumst vestibulum rhoncus est pellentesque. Eu non diam phasellus vestibulum lorem sed risus. Semper risus in hendrerit gravida. Pharetra massa massa ultricies mi quis hendrerit dolor magna. Urna id volutpat lacus laoreet. Lacus suspendisse faucibus interdum posuere lorem. Mauris pharetra et ultrices neque ornare. Sed lectus vestibulum mattis ullamcorper velit sed. Non odio euismod lacinia at. In mollis nunc sed id. Molestie ac feugiat sed lectus vestibulum mattis ullamcorper velit. Gravida cum sociis natoque penatibus et magnis. Tincidunt dui ut ornare lectus. Integer quis auctor elit sed.
+
```
Comment (gh-team-member) [inline external/feature.md:2]: c2

**Review by gh-team-member — CHANGES_REQUESTED** [review]
please make the suggested changes.

**Pull request merged**